python/lsst/afw/table/_syntax.py

```diff
@@ -202,3 +202,70 @@ def processArray(a):
         else:
             d[name] = processArray(self.get(schemaItem.key))
     return d
+
+def BaseCatalog_asAstropy(self, cls=None, copy=False, unviewable="copy"):
+    """!
+    Return an astropy.table.Table (or subclass thereof) view into this catalog.
+
+    @param[in]   cls        Table subclass to use; None implies astropy.table.Table itself.
+                            Use astropy.table.QTable to get Quantity columns.
+
+    @param[in]  copy        Whether to copy data from the LSST catalog to the astropy table.
+                            Not copying is usually faster, but can keep memory from being
+                            freed if columns are later removed from the Astropy view.
+
+    @param[in]  unviewable  One of the following options, indicating how to handle field types
```

Comment:
Since this is the first case of using `__astropy_table__` and it looks like passing through `**kwargs` would be useful, this might indicate it will be useful in other cases.  This is already officially part of the `__astropy_table__` API (except there is nothing to populate `**kwargs`).  We would need to have some discussion, but it isn't entirely unreasonable to have `**kwargs` in the main `Table` API for this purpose.  I would be much happier to have `Table(lsst_table, unviewable='raise')` be the one true way to convert, rather than one mostly-works method and another fully-works method.